README.rst

```diff
@@ -44,57 +41,58 @@ For example::
 
-To get an optimized build of Python, ``configure --enable-optimizations`` before
-you run ``make``.  This sets the default make targets up to enable Profile Guided
-Optimization (PGO) and may be used to auto-enable Link Time Optimization (LTO)
-on some platforms.  For more details, see the sections bellow.
+To get an optimized build of Python, ``configure --enable-optimizations``
+before you run ``make``.  This sets the default make targets up to enable
+Profile Guided Optimization (PGO) and may be used to auto-enable Link Time
+Optimization (LTO) on some platforms.  For more details, see the sections
+below.
 
 
 Profile Guided Optimization
 ---------------------------
 
-PGO takes advantage of recent versions of the GCC or Clang compilers.
-If ran, ``make profile-opt`` will do several steps.
+PGO takes advantage of recent versions of the GCC or Clang compilers.  If ran,
+``make profile-opt`` will do several steps.
 
-First, the entire Python directory is cleaned of temporary files that
-may have resulted in a previous compilation.
+First, the entire Python directory is cleaned of temporary files that may have
+resulted in a previous compilation.
 
 Then, an instrumented version of the interpreter is built, using suitable
-compiler flags for each flavour. Note that this is just an intermediary
-step and the binary resulted after this step is not good for real life
-workloads, as it has profiling instructions embedded inside.
+compiler flags for each flavour. Note that this is just an intermediary step
+and the binary resulted after this step is not good for real life workloads, as
+it has profiling instructions embedded inside.
 
-After this instrumented version of the interpreter is built, the Makefile
-will automatically run a training workload. This is necessary in order to
-profile the interpreter execution. Note also that any output, both stdout
-and stderr, that may appear at this step is suppressed.
+After this instrumented version of the interpreter is built, the Makefile will
+automatically run a training workload. This is necessary in order to profile
+the interpreter execution. Note also that any output, both stdout and stderr,
+that may appear at this step is suppressed.
 
 Finally, the last step is to rebuild the interpreter, using the information
-collected in the previous one. The end result will be a Python binary
-that is optimized and suitable for distribution or production installation.
+collected in the previous one. The end result will be a Python binary that is
+optimized and suitable for distribution or production installation.
 
 
 Link Time Optimization
 ----------------------
 
 Enabled via configure's ``--with-lto`` flag.  LTO takes advantages of recent
-compiler toolchains ability to optimize across the otherwise arbitrary ``.o`` file
-boundary when building final executables or shared libraries for additional
-performance gains.
+compiler toolchains' ability to optimize across the otherwise arbitrary ``.o``
```

Comment:
yep, because I have not seen the possessive form, but just a mistake in the text.